inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/cmd/CommandExecutorImpl.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.service.cmd;
+
+import org.apache.inlong.manager.common.consts.InlongConstants;
+import org.apache.inlong.manager.pojo.cluster.agent.AgentClusterNodeRequest;
+import org.apache.inlong.manager.service.cmd.shell.ShellExecutorImpl;
+import org.apache.inlong.manager.service.cmd.shell.ShellTrackerImpl;
+
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+public class CommandExecutorImpl implements CommandExecutor {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CommandExecutorImpl.class);
+
+    private String join(List<String> strings) {
+        StringBuilder buffer = new StringBuilder();
+        for (String string : strings) {
+            buffer.append(string).append(InlongConstants.NEW_LINE);
+        }
+        return buffer.toString();
+    }
+
+    private String join(String... strings) {
+        StringBuffer buffer = new StringBuffer();
+        for (String string : strings) {
+            buffer.append(string).append(InlongConstants.BLANK);
+        }
+        return buffer.toString();
+    }
```

Comment:
JDK already provide the join function like `String.join(CharSequence delimiter, CharSequence... elements) `

Reply:
Done.